### PR TITLE
chore: Update devfile schema from 2.0.0 to 2.2.2

### PR DESCRIPTION
**Pull request by creydr**
## What is the purpose of this change? What does it change?

Currently an old devfile schema is used. This leads to schema validation warnings like in the following:

image

This is address in this issue and the schema updated to the latest version (2.2.2): https://github.com/devfile/api/blob/v2.2.2/schemas/latest/devfile.json

## Was the change discussed in an issue?
no

### CODE REVIEW

File: src/main/resources/schemas/devfile.json [hunk 1]
```diff
@@ -144,7 +141,7 @@
             ],
             "properties": {
               "commandLine": {
-                "description": "The actual command-line string\n\nSpecial variables that can be used:\n\n - `$PROJECTS_ROOT`: A path where projects sources are mounted as defined by container component's sourceMapping.\n\n - `$PROJECT_SOURCE`: A path to a project source ($PROJECTS_ROOT/\u003cproject-name\u003e). If there are multiple projects, this will point to the directory of the first one.",
+                "description": "The actual command-line string\n\nSpecial variables that can be used:\n\n - `$PROJECTS_ROOT`: A path where projects sources are mounted as defined by container component's sourceMapping.\n\n - `$PROJECT_SOURCE`: A path to a project source ($PROJECTS_ROOT/<project-name>). If there are multiple projects, this will point to the directory of the first one.",
```
Comment (sbouchet) [inline src/main/resources/schemas/devfile.json:144]: original json uses \u003c, can you also uses it ? 
https://github.com/devfile/api/blob/a6ec0a38307b63a29fad2eea945cc69bee97a683/schemas/latest/devfile.json#L144

Comment (creydr) [reply src/main/resources/schemas/devfile.json]: Uh. Sorry, I don't know why this happened during the copy. Updated it

**Review by sbouchet — APPROVED** [review]
LGTM

**Pull request merged**